apps/crn-server/src/routes/research-outputs.route.ts

```diff
@@ -46,16 +53,21 @@ export const researchOutputRouteFactory = (
   );
 
   researchOutputRoutes.post('/research-outputs', async (req, res) => {
-    const { body, loggedInUser } = req;
+    const { body, loggedInUser } = req as ResearchOutputPostRequest;
```

Comment:
Can we avoid the "as", for example doing something like:

researchOutputRoutes.post<ResearchOutputPostRequest>(